stubs/aiofiles/aiofiles/threadpool/text.pyi

```diff
@@ -37,3 +37,5 @@ class AsyncTextIOWrapper(AsyncBase[str]):
     def name(self) -> FileDescriptorOrPath: ...
     @property
     def mode(self) -> str: ...
+
+class AsyncTextIndirectIOWrapper(AsyncIndirectBase[str], AsyncTextIOWrapper): ...
```

Comment:
Similar comment to the one I made about AsyncIndirectBufferedIOBase -- I'm wary of inheriting from `AsyncTextIOWrapper` here.

Reply:
This one inherits a lot of methods. I added _UnknownAsyncTextIO to have methods defined once and have both public text io's inherit from that. The mro now for public classes matches runtime.

If you'd prefer to not use that solution I'll do duplication way similar to others.

Reply:
The solution you have looks good